### PR TITLE
GVRP instances

### PR DESCRIPTION
**Pull request by N-Wouda**
Part of https://github.com/PyVRP/PyVRP/issues/470.

### CODE REVIEW

**Comment by N-Wouda** [comment]
BKS are from a 1h run.

File: GVRP/M-n200-k17-C100-V8.gvrp
Comment (leonlan) [inline GVRP/M-n200-k17-C100-V8.gvrp:1]: What's the difference between this instance and `M-n200-k16-C100-V8`? I don't see this one in the original raw data.

File: GVRP/G-n262-k25-C131-V12.gvrp
Comment (leonlan) [inline GVRP/G-n262-k25-C131-V12.gvrp:1]: Let's also rename the file extension to `.vrp` to be consistent with the other instances

File: GVRP/M-n200-k17-C100-V8.gvrp
Comment (N-Wouda) [reply GVRP/M-n200-k17-C100-V8.gvrp:1]: This was in the data dump I received, but it does seem to be a duplicate. I've removed it.

**Pull request merged**